src/Valleysoft.DockerCredsProvider.Test/CredsProviderTests.cs

```diff
@@ -9,11 +9,19 @@ namespace Valleysoft.DockerCredsProvider.Test;
 
 public class CredsProviderTests
 {
+    IEnvironment DefaultEnvironmentMock;
```

Comment:
I don't see any definition of `IEnvironment` or `Environment` in these changes.

Reply:
unstaged file - pushed it now.